ui/apps/platform/src/Containers/Integrations/IntegrationsPage.tsx

```diff
@@ -1,17 +1,58 @@
 import type { ReactElement } from 'react';
-import { Route, Routes } from 'react-router-dom-v5-compat';
+import { Navigate, Route, Routes } from 'react-router-dom-v5-compat';
+
+import useCentralCapabilities from 'hooks/useCentralCapabilities';
+import useFeatureFlags from 'hooks/useFeatureFlags';
+import type { IntegrationSource } from 'types/integration';
 
 import IntegrationsNotFoundPage from './IntegrationsNotFoundPage';
-import IntegrationTilesPage from './IntegrationTiles/IntegrationTilesPage';
 import IntegrationsListPage from './IntegrationsListPage';
 import CreateIntegrationPage from './CreateIntegrationPage';
 import EditIntegrationPage from './EditIntegrationPage';
 import IntegrationDetailsPage from './IntegrationDetailsPage';
 
+import AuthenticationIntegrationsTab from './IntegrationTiles/AuthenticationIntegrationsTab';
+import BackupIntegrationsTab from './IntegrationTiles/BackupIntegrationsTab';
+import CloudSourceIntegrationsTab from './IntegrationTiles/CloudSourceIntegrationsTab';
+import ImageIntegrationsTab from './IntegrationTiles/ImageIntegrationsTab';
+import NotifierIntegrationsTab from './IntegrationTiles/NotifierIntegrationsTab';
+import SignatureIntegrationsTab from './IntegrationTiles/SignatureIntegrationsTab';
+import type { IntegrationsTabElement } from './IntegrationTiles/IntegrationsTab.types';
+
+import { getSourcesEnabled } from './utils/integrationsList';
+
+// Adapted from routeComponentMap from Body.tsx file.
+
+const integrationsTabElementMap: Record<IntegrationSource, IntegrationsTabElement> = {
+    imageIntegrations: ImageIntegrationsTab,
+    signatureIntegrations: SignatureIntegrationsTab,
+    notifiers: NotifierIntegrationsTab,
+    backups: BackupIntegrationsTab,
+    cloudSources: CloudSourceIntegrationsTab,
+    authProviders: AuthenticationIntegrationsTab,
+};
+
 const IntegrationsPage = (): ReactElement => {
+    const { isCentralCapabilityAvailable } = useCentralCapabilities();
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+    const sourcesEnabled = getSourcesEnabled({
+        isCentralCapabilityAvailable,
+        isFeatureFlagEnabled,
+    });
+
     return (
         <Routes>
-            <Route index element={<IntegrationTilesPage />} />
+            <Route index element={<Navigate to={sourcesEnabled[0]} />} />
+            {sourcesEnabled.map((source) => {
+                const Element = integrationsTabElementMap[source];
```

Comment:
Nit, because this is well-typed and unlikely to be a problem. The de-coupling of runtime values for `integrationsTabElementMap` and `sourcesEnabled` _could_ mean a future type change to either would open the door for `Element` to be `undefined`, which could result in an error during rendering.

Thoughts on checking if `Element` is truthy and returning `null` otherwise? Is that overkill?